docs/sources/clients/promtail/configuration.md

```diff
@@ -1622,6 +1626,116 @@ users with thousands of services it can be more efficient to use the Consul API
 directly which has basic support for filtering nodes (currently by node
 metadata and a single tag).
 
+### docker_sd_config
+
+Docker service discovery configurations allow retrieving targets from a Docker daemon.
+It will only watch containers of the daemon referenced with the host parameter. That
+means it should run on each node in a distributed setup. The containers must run with
+either the [json-file](https://docs.docker.com/config/containers/logging/json-file/)
+or [journald](https://docs.docker.com/config/containers/logging/journald/) logging driver.
```

Comment:
Trying here to suggest wording that clarifies "it" in more than one place. If my clarifications are wrong, please revise to clarify correctly.

```suggestion
Docker service discovery allows retrieving targets from a Docker daemon.
It will only watch containers of the Docker daemon referenced with the host parameter. Docker
service discovery should run on each node in a distributed setup. The containers must run with
either the [json-file](https://docs.docker.com/config/containers/logging/json-file/)
or [journald](https://docs.docker.com/config/containers/logging/journald/) logging driver.
```